Schedule.API/Services/Procedures/EquipmentRelocationSchedulingService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using General.Repository;
+using Schedule.API.DTOs;
+using Schedule.API.Infrastructure.Repositories.Procedures.Interfaces;
+using Schedule.API.Model.Procedures;
+using Schedule.API.Services.Procedures.Interface;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Schedule.API.Services.Procedures
+{
+    public class EquipmentRelocationSchedulingService : IEquipmentRelocationSchedulingService
+    {
+        private readonly RepositoryWrapper<IExaminationRepository> _examinationWrapper;
+
+        public EquipmentRelocationSchedulingService(IExaminationRepository examinationRepository)
+        {
+            this._examinationWrapper = new RepositoryWrapper<IExaminationRepository>(examinationRepository);
+
+        }
+
+        public IEnumerable<int> CheckRoomsAvailability(EquipmentRelocationDto eqRealDto)
+        {
```

Comment:
Change name of method. For me this name sounds like some bool method, but it returns int list.

Reply:
I renamed it. Now it is called GetUnavailableRooms